docs/source/features/peer.rst

```diff
@@ -0,0 +1,38 @@
+.. _rst_gdeploypeer:
+
+Peer
+^^^^
+
+*peer* module allows user to add/probe remote machines to a Trusted Storage Pool,
+detach/remove a remote machine from a Trusted Storage Pool.
+action variable can be any of *probe*, *detach*, or *ignore*.
+
```

Comment:
@devyanikota action supports only `probe' or `detach'

ignore_errors is a separate variable altogether. Can you please correct that?